apps/Makefile.am

```diff
@@ -3,7 +3,7 @@
 
 include $(top_srcdir)/Makefile.incl
 
-exampledir = $(prefix)/lib/boinc-apps-examples
+exampledir = $(libdir)/boinc-apps-examples
```

Comment:
Doesn't this mean that after your change there is no way to use custom location for the installation?